src/test/java/seedu/anilist/model/anime/NameTest.java

```diff
@@ -26,22 +38,18 @@ public void isValidName() {
         assertThrows(NullPointerException.class, () -> Name.isValidName(null));
 
         // invalid name
-        assertFalse(Name.isValidName("")); // empty string
-        assertFalse(Name.isValidName(" ")); // spaces only
-        assertFalse(Name.isValidName(" Danshi Koukousei no Nichijou")); // starting with space
-        assertFalse(Name.isValidName(VALID_NAME_MAX_LENGTH + "A")); // exceed max length
+        assertFalse(Name.isValidName(INVALID_STRING_EMPTY));
+        assertFalse(Name.isValidName(INVALID_STRING_SPACE));
+        assertFalse(Name.isValidName(INVALID_STRING_NON_ASCII));
+        assertFalse(Name.isValidName(INVALID_NAME_LONGER_THAN_MAX_LENGTH));
 
         // valid name
-        assertTrue(Name.isValidName("Danshi Koukousei no Nichijou")); // alphabets only
-        assertTrue(Name.isValidName("12345")); // numbers only
-        assertTrue(Name.isValidName(":;.'!")); // punctuation only
-        assertTrue(Name.isValidName("~!@#$%^&()+-=:;")); // non-alphanumeric ascii characters
-        assertTrue(Name.isValidName("3-gatsu no Lion")); // starting with numeric character containing punctuation
-        assertTrue(Name.isValidName("No. 6")); // starting with alphabet containing punctuation
-        assertTrue(Name.isValidName("BLEACH")); // with all capital letters
-        // long names
-        assertTrue(Name.isValidName("Higehiro: After Being Rejected, I Shaved and Took in a High School Runaway"));
-        assertTrue(Name.isValidName(VALID_NAME_MAX_LENGTH)); // max length
-        assertTrue(Name.isValidName("Gintama'")); // name containing apostrophe
+        assertTrue(Name.isValidName(VALID_NAME_BNHA));
+        assertTrue(Name.isValidName(VALID_NAME_AKIRA));
+        assertTrue(Name.isValidName(VALID_NAME_EIGHTY_SIX));
+        assertTrue(Name.isValidName(VALID_NAME_FATE_ZERO));
+        assertTrue(Name.isValidName(VALID_NAME_JOJO_GOLDEN_WIND));
+        assertTrue(Name.isValidName(VALID_NAME_NON_ALPHANUMERIC_ASCII));
+        assertTrue(Name.isValidName(VALID_NAME_MAX_LENGTH));
```

Comment:
might want to add comments to make equivalence partitioning more clear
either add in each test file, or add it inside command test utils

Reply:
ok